flycheck.el

```diff
@@ -9350,6 +9350,11 @@ See URL `http://pear.php.net/package/PHP_CodeSniffer/'."
             ;; been configured with show_progress enabled
             "-q"
             (option "--standard=" flycheck-phpcs-standard concat)
+            ;; Some files are not detected correctly
+            ;; so it is necessary to pass the extension.
+            (eval
+             (when (file-name-extension (buffer-file-name))
```

Comment:
You need to condition on buffer-file-name as well, since it may be nil.

Something like 

```
(-when-let ((fname buffer-file-name)
            (ext (file-name-extension buffer-file-name))) 
  …)
```

Reply:
Umm, what do you think about using (when (buffer-file-name).. like the below option?

Reply:
Sure, that's what the change I suggested does.